test/test_ops.py

```diff
@@ -166,13 +166,15 @@ def test_neg(self):
   def test_mul(self):
     helper_test_op([(64,64), (64,64)], lambda x,y: x*y, Tensor.mul)
     helper_test_op([(), ()], lambda x,y: x*y, Tensor.mul)
+  @unittest.skipIf(getenv("CI", "") != "" and Device.DEFAULT == "METAL", "broken in METAL CI")
```

Comment:
These need to be fixed, I don't see why they should be broken. I'm fine with skipping ViT

Reply:
looks like they are passing now, just have to separate them out

Reply:
I discussed with @cloud11665 since he wrote the tests. He believes the test fail is caused by the architectural difference between the iGPU (in CI runners) vs the dGPUs I'm testing on.